src/components/TwoLevelSelector.js

```diff
@@ -0,0 +1,377 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+const Radium = require('radium');
+
+const Icon = require('./Icon');
+const StyleConstants = require('../constants/Style');
+
+const TwoLevelSelector = React.createClass({
+
+  propTypes: {
+    data: React.PropTypes.object,
+    onChange: React.PropTypes.func,
+    selectedItem: React.PropTypes.string,
```

Comment:
Okay, I will change it. The structure is the same but the naming of properties is little different.
```javascript
[
  {
    label: 'groceries',
    value: '1xxxx',
    items: [
      {
        label: 'milk',
        value: '12xxx'
      },
      {
        label: 'eggs',
        value: '23xxx'
      }
    ]
  },
  {
    label: 'books',
    value: '31xxx',
    items: [
      {
        label: 'bookXY',
        value: '31xxx'
      }
    ]
  }
]
```